Makefile

```diff
@@ -100,13 +100,13 @@ start-docs: ## Start the local mkdocs based development environment.
 
 .PHONY: clean-docs
 clean-docs: ## Remove all local mkdocs Docker images (cleanup).
-	docker container prune --force --filter "label=project=virtlet_documentation"
+	docker container prune --force --filter "label=project=libvirt-provider_documentation"
 
 ##@ Build
 
 .PHONY: build
 build: generate fmt add-license lint ## Build the binary
-	GOOS=$(TARGET_OS) GOARCH=$(TARGET_ARCH) go build -o bin/virtlet ./main.go
+	GOOS=$(TARGET_OS) GOARCH=$(TARGET_ARCH) go build -o $(LIBVERTPROVIDERBIN) ./main.go
 
 .PHONY: run
 run-base: generate fmt lint ## Run the binary
```

Comment:
pls fix path for go run too, it is invalid

Reply:
corrected the path and tested make build is running fine.

Reply:
i asked you in this thread for fix path in run-base target

Reply:
Corrected the path for make run as well..